docs/conf/portland/2020/news/covid-19-update-01.rst

```diff
@@ -0,0 +1,19 @@
+:template: {{year}}/generic.html
+
+.. post:: March 3, 2020
+   :tags: portland-2020, covid-19
+
+COVID-19 Update
+===============
+
+Hey Folks,
+
+As you've probably heard by now, the coronavirus (also known as COVID-19) is a new virus that causes respiratory illness in people and can easily spread from person-to-person. Since Write the Docs Portland 2020 is only a few months away, we wanted to give you an update on our status regarding COVID-19.
+
+**As of today, March 3, Write the Docs Portland 2020 is scheduled to go ahead**.
+
+We are watching the situation closely, and taking advice from the Oregon health authorities and the United States CDC (Center for Disease Control). We plan to reassess the situation weekly, you can find updates tagged with `COVID-19 </blog/archive/tag/covid-19/>`_.
```

Comment:
```suggestion
We are watching the situation closely, and taking advice from the Oregon health authorities and the United States CDC (Centers for Disease Control and Prevention). We plan to reassess the situation weekly, you can find updates tagged with `COVID-19 </blog/archive/tag/covid-19/>`_.
```

Reply:
CDCP did you say? @ddbeck :japanese_ogre:

Reply:
Hey, of all the weird things about US government, this initialism mismatch is the least weird